package.json

```diff
@@ -23,7 +23,8 @@
   "dependencies": {
     "@avcs/autosuggest": "^1.7.5",
     "@floip/expression-evaluator": "^2.1.0",
-    "@floip/flow-runner": "1.0.0-rc2.7",
+    "@floip/expression-parser": "^1.11.3",
+    "@floip/flow-runner": "https://github.com/FLOIP/flow-runner.git#feature/VMO-5090/validate-expressions",
```

Comment:
@object-required , I think we may have a version conflict here. In fact,
- you branch flow-runner from `master`, right ?
- I already merged flow-runner rc3 into master https://github.com/FLOIP/flow-runner/pull/67, I didn't know we will have flow-runner for your ticket
- Which means, if we run `yarn install` in flow-builder, it will consume flow-runner `rc3`. And will will have a TS error about `Property 'config' does not exist on type 'IBlockExit'.` in flow-builder.

Trying to resolve this could be a bit tricky at this stage, so I'd like to suggest we consume `rc4` in your tickets, meaning:
- base flow-runner from this branch https://github.com/FLOIP/flow-runner/pull/68  instead of master
- base flow-builder from this branch https://github.com/FLOIP/flow-builder/pull/219 instead of master

Anyway, those rc4 PRs are already approved, moving them to QA

Reply:
@object-required , After QA, and you merged flow-runner, make sure you use semantic versioning from npm here `^1.0.0-rc4` ;)